### PR TITLE
Final tweaks

### PR DESCRIPTION
**Pull request by iHiD**
Each commit has a description.

### CODE REVIEW

File: exercises/ruby-1-b/ruby_1_b.rb [hunk 1]
```diff
@@ -4,52 +4,12 @@ class Library
   CATALOG_DATA = <<~DATA
     The Adventures of Tom Sawyer;9780191604928;Mark Twain;2007
     Republic;9780718198916;Plato;2012
-    Programming Ruby: The Pragmatic Programmers' Guide;9780974514055;David Thomas;2004
+    Programming Ruby: The Pragmatic Programmers Guide;9780974514055;David Thomas;2004
```
Comment (iHiD) [inline exercises/ruby-1-b/ruby_1_b.rb]: Our code editor can't cope with syntax highlighting if this is here, so I've just removed it.

File: exercises/ruby-1-b/README.md [hunk 2]
```diff
@@ -6,88 +6,66 @@ each book has the following columns:
...
 ```
 
-and `lookup_publication_year` returns the publication year
+The library then has methods that need adding to handle stock.
 
-```ruby
-library.lookup_publication_year("9781986431484")
-=> "2018"
-```
+- `add_stock!` increases the stock count of a book
```
Comment (ErikSchierboom) [inline exercises/ruby-1-b/README.md]:
```suggestion
- `add_stock!` increases the stock count of a book.
```

File: exercises/ruby-1-b/README.md [hunk 3]
```diff
@@ -6,88 +6,66 @@ each book has the following columns:
...
+- `borrow!` decreases the stock count of a book if there are books in stock.
+- `put_back!` does the opposite of `borrow!`. It increases the stock if any of the books are still borrowed.
+- `lookup_stock` returns the current stock of a book
```
Comment (ErikSchierboom) [inline exercises/ruby-1-b/README.md]:
```suggestion
- `lookup_stock` returns the current stock of a book.
```

File: exercises/ruby-1-b/README.md [hunk 4]
```diff
@@ -6,88 +6,66 @@ each book has the following columns:
...
+- `book_in_stock?` returns a boolean representing whether there is at least 1 book in stock.
+- `books_in_stock` lists the ISBNs of all books in stock, sorted alphabetically
```
Comment (ErikSchierboom) [inline exercises/ruby-1-b/README.md]:
```suggestion
- `books_in_stock` lists the ISBNs of all books in stock, sorted alphabetically.
```

File: exercises/ruby-1-b/README.md [hunk 5]
```diff
@@ -6,88 +6,66 @@ each book has the following columns:
...
+- `total_books_in_stock` returns the summed stock of all books
```
Comment (ErikSchierboom) [inline exercises/ruby-1-b/README.md]:
```suggestion
- `total_books_in_stock` returns the summed stock of all books.
```

**Comment by iHiD** [comment]
Thanks @ErikSchierboom :)

**Pull request merged**